lib/matplotlib/table.py

```diff
@@ -453,16 +494,27 @@ def get_celld(self):
         'return a dict of cells in the table'
         return self._cells
 
+    def get_cell_type(self):
+        return self._cellType
+
+    def set_cell_type(self, cellType):
+        if cellType in ('default', 'scicell'):
+            self._cellType = cellType
+
+        else:
+            raise ValueError('Unrecognized cell type %s; '
+                'Has to be default or scicell' % cellType)
+
 
 def table(ax,
-    cellText=None, cellColours=None,
+    cellType=None, cellText=None, cellColours=None,
```

Comment:
This has to go last to maintain back-compatibility for anyone who has been using this with positional arguments.